taskcall/manifest.json

```
@@ -0,0 +1,71 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "dd54da03-0a8c-4796-aaa6-61eeb04e611b",
+  "app_id": "taskcall",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor and centralize Datadog incidents with TaskCall.",
+    "title": "TaskCall",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Incident Notifications",
+        "image_url": "images/incident_notifications.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Incident Details",
+        "image_url": "images/incident_details.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Impact Visibility and Status Dashboard",
+        "image_url": "images/impact_visibility_status_dashboard.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "On-Call Management",
+        "image_url": "images/on_call_management.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Alerting",
+      "Category::Collaboration",
+      "Category::Incidents",
+      "Category::Issue Tracking",
+      "Category::Notification",
+      "Offering::UI Extension",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "integration": {
+      "source_type_name": "TaskCall",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "taskcall.",
+        "check": [],
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    }
+  },
```

Is your integration sending in events? 

If not, because this is an API-based integration, you can remove the integration definition here and also delete the `service_checks.json` and `metadata.csv` files
```suggestion
  "assets": {},
```